uber/templates/accounts/index.html

```diff
@@ -67,8 +67,8 @@
         <td>
             <form class="form update-form" method="post" action="update">
                 <input type="hidden" name="id" value="{{ account.id }}" />
-                {% csrf_token %}
-                <div class="form-group">{% checkgroup account.access %}</div>
+                {{ csrf_token() }}
+                <div class="form-group">{{ account.html_checkgroup('access') }}</div>
```

Comment:
As per my earlier comment, I'd prefer to see this as ``{{ checkgroup(account.access) }}`` though I'm willing to look the other way, especially in the sense of getting something working before we launch.